### DISCUSSION TITLE
CameraPathUI

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#1057] [Q&A] [answered]
in CameraPathUI demo,

CameraPath class  
let result = getPointAndDirection(_fraction_: number)  

now if i  give the result  value how get the  _fraction_ value ?

**Comment by pmconne** [comment, marked as answer]
@EarlinLutz inverse of `CurveChainWithDistanceIndex.fractionToPoint`?
[Sample in question](https://www.itwinjs.org/sample-showcase/?group=Viewer+Features&sample=Camera+Path+Sample&imodel=Metrostation+Sample).

**Reply by paprophet** [reply]
yes ,inverse of CurveChainWithDistanceIndex.fractionToPoint

**Reply by EarlinLutz** [reply]
That's         curve.closestPoint       to get the fraction.
